src/api/app/controllers/webui/packages/meta_controller.rb

```diff
@@ -12,6 +12,8 @@ class MetaController < Webui::WebuiController
       after_action :verify_authorized, only: :update
 
       def show
+        return if @project.scmsync.present?
```

Comment:
Return isn't how you handle a on existing object in a route right?

Reply:
Right, please take a look again.